<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistrSetDialog</class>
 <widget class="QDialog" name="DistrSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор статистических распределений</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Доступные распределения:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="lstAvailable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnTakeOutAll">
         <property name="text">
          <string>Убрать все</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTakeOut">
         <property name="text">
          <string>Убрать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddAll">
         <property name="text">
          <string>Добавить все</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Используемые распределения:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="lstApplied">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Метод оценки параметров</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="cbxMD">
          <property name="text">
           <string>Минимального расстояния</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxMLH">
          <property name="text">
           <string>Максимального правдоподобия</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxSimpleMLH">
          <property name="text">
           <string>Максимального правдоподобия (простой)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Критерии однородности</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="cbxKolmogorov">
          <property name="text">
           <string>Колмогорова</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxCramerVonMises">
          <property name="text">
           <string>Крамера-Мизеса (омега-2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxAndersonDarling">
          <property name="text">
           <string>Андерсона-Дарлинга (Омега-2)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBox</sender>
   <signal>accepted()</signal>
   <receiver>DistrSetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBox</sender>
   <signal>rejected()</signal>
   <receiver>DistrSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTakeOutAll</sender>
   <signal>clicked()</signal>
   <receiver>DistrSetDialog</receiver>
   <slot>slotManageLists()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTakeOut</sender>
   <signal>clicked()</signal>
   <receiver>DistrSetDialog</receiver>
   <slot>slotManageLists()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>DistrSetDialog</receiver>
   <slot>slotManageLists()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddAll</sender>
   <signal>clicked()</signal>
   <receiver>DistrSetDialog</receiver>
   <slot>slotManageLists()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotManageLists()</slot>
 </slots>
</ui>
